packages/core/src/node/backend-application.ts

```diff
@@ -73,9 +73,9 @@ export class BackendApplication {
         // Handles normal process termination.
         process.on('exit', () => this.onStop());
         // Handles `Ctrl+C`.
-        process.on('SIGINT', () => this.onStop());
+        process.on('SIGINT', () => process.exit(0));
```

Comment:
What is the default if we remove it? The same question for `SIGTERM`.

Reply:
It exits with no exit event so no opportunity to call onstop

Reply:
ok